src/Microsoft.ML.Data/Training/TrainerEstimatorBase.cs

```diff
@@ -60,7 +60,6 @@ public abstract class TrainerEstimatorBase<TTransformer, TModel> : ITrainerEstim
             Contracts.CheckValue(host, nameof(host));
             Host = host;
             Host.Check(feature.IsValid, nameof(feature));
-            Host.Check(label.IsValid, nameof(label));
```

Comment:
We remove this because in Kmeans, `label` is just default so that `IsValid` is `false`.